app/src/androidTest/java/com/example/parkingsystem/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.parkingsystem;
+
+import android.content.Context;
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        assertEquals("com.example.parkingsystem", appContext.getPackageName());
+    }
+}
```

Comment:
to fix this you should apply File > Preferences > Editor > General > Ensure line feed at file end on Save at Android Studio

Reply:
Applied! Thank you!